hypermedia-pipeline/src/html/fetch-markdown.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const client = require('request-promise');
+
```

Comment:
it's quite hard to read.. maybe:

- dereference `REPO_RAW_ROOT` asa different name. it looks awkward in the code below
- use a nice `function` instead of the arrow and decouple it from module.exports
- omit the `= {}` default argument specifier in order to enforce passing an object.